AccountantApp/src/test/java/com/tap/project/escrivaghera/AccountantApp/IT/AccountantAppIT.java

```diff
@@ -36,50 +35,53 @@ public void setUp() throws Exception {
 
 	@Test
 	public void testAddIT() {
-		myAccountantApp.authenticate(myUser);
-		Date[] dates = createDates();
-		JournalEntry testEntry = myAccountantApp.createJournalEntry("1", dates[0], createTestList(1100, 1100));
+		Date[] dates = authenticateAndCreateDates();
+		JournalEntry testEntry = myAccountantApp.createJournalEntry("1", dates[0],
```

Comment:
(scrivo in italiano per una volta)

Tipo qui, non posso chiamare myAccountantApp.createJournalEntry, perchè se questo metodo non funziona per qualche motivo poi non mi funziona più il test, invece se questa parte la faccio con il mongoTestHelper quello è fuori da questa classe e è sicuro che funziona e non viene modificato da nessuno. Ma non solo qui, anche myDB.add qui sotto non devo chiamarlo, devo fare con l'add del MongoTestHelper, e così per ogni test su questa classe, perchè è un IT....invece nel test unitario della classe va bene usare solo il genericHelper. Ti torna?